Rule/Rule/Base.ts

```diff
@@ -13,14 +13,20 @@ export interface Base {
 export namespace Base {
 	export const kinds = ["authorization", "outbound", "inbound"] as const
```

Comment:
I think we should declare constants in their namespaces, exported as `<namespace>.values`

Reply:
\+